dotcom-rendering/src/components/Lightbox.stories.tsx

```diff
@@ -324,7 +299,30 @@ export const WithLabs = () => {
 };
 
 /**
- * with html in caption (links)
- * with a portrait image
- *
+ * We toggle the [i] button to close and open the caption
  */
+export const ClickInfo = () => {
+	const format = {
+		display: ArticleDisplay.Standard,
+		design: ArticleDesign.Standard,
+		theme: Pillar.News,
+	};
+	const images = [{ ...testImage, title: 'ClickInfoTest' }];
+	return (
+		<Initialise>
+			<LightboxLayout format={format} images={images} />
+		</Initialise>
+	);
+};
+ClickInfo.play = async ({ canvasElement }: { canvasElement: HTMLElement }) => {
+	storage.local.clear();
+	const canvas = within(canvasElement);
+
+	await canvas.findByText(/ClickInfoTest/);
```

Comment:
This is the trick to ensuring interaction tests work correctly. First, we wait for the expected element to be visible on screen with `findBy...` https://testing-library.com/docs/queries/about/#types-of-queries

Then, we can set expectations on behaviour.

If we don't first wait for the page to be in the correct state (we do some setup in [LightboxJavascript](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/components/LightboxJavascript.tsx)), then the test may fail.